tests/secrets/list.test.ts

```diff
@@ -41,124 +40,121 @@ describe('commandListSecrets', () => {
     });
   });
 
-  test(
-    'should fail when vault does not exist',
-    async () => {
-      command = ['secrets', 'ls', '-np', dataDir, 'DoesntExist'];
-      const result = await testUtils.pkStdio([...command], {
-        env: {
-          PK_PASSWORD: password,
-        },
-        cwd: dataDir,
-      });
-      expect(result.exitCode).toBe(64); // Sysexits.USAGE
-    },
-    globalThis.defaultTimeout * 2,
-  );
-
-  test(
-    'should list secrets',
-    async () => {
-      const vaultName = 'Vault4' as VaultName;
-      const vaultId = await polykeyAgent.vaultManager.createVault(vaultName);
-
-      await polykeyAgent.vaultManager.withVaults([vaultId], async (vault) => {
-        await vaultOps.addSecret(vault, 'MySecret1', '');
-        await vaultOps.addSecret(vault, 'MySecret2', '');
-        await vaultOps.addSecret(vault, 'MySecret3', '');
-      });
-
-      command = ['secrets', 'ls', '-np', dataDir, vaultName];
-      const result = await testUtils.pkStdio([...command], {
-        env: {
-          PK_PASSWORD: password,
-        },
-        cwd: dataDir,
-      });
-      expect(result.exitCode).toBe(0);
-      expect(result.stdout).toBe('MySecret1\nMySecret2\nMySecret3\n');
-    },
-    globalThis.defaultTimeout * 2,
```

Comment:
When I ran the tests, I saw no need for additional timeout, so I removed this line. This applies to all secrets subcommand tests, as most of them completed within 3 seconds, with longer ones taking up to 5, which is well under the 20 second timeout limit.

Reply:
Usually we preserve the `globalThis.defaultTimeout` because that's the default.